### ISSUE TITLE
Less compilation problems in Magento/backend theme

### ISSUE THREAD

**Issue by hostep** [magento/magento2#24004]
### Description

This is similar issue as what was being reported in https://github.com/magento/magento2/issues/23619
But this time it's for the Magento/backend theme.
The fact that there are errors in the less code might indicate that certain things aren't getting outputted in css properly. But without deeper research I can't confirm this yet.

### Preconditions (*)
1. Magento 2.3-develop branch (or any older released version)
2. NodeJs (I'm using v8.16.0)
3. PHP 7.2

### Steps to reproduce (*)
1. Setup a vanilla Magento installation
2. Run `npm install less@3.9.0`
3. Run `rm -R var/view_preprocessed/* pub/static/*; bin/magento setup:static-content:deploy -f --theme=Magento/backend en_US`
4. Run `node_modules/.bin/lessc -l var/view_preprocessed/pub/static/adminhtml/Magento/backend/en_US/css/styles-old.less`

### Expected result (*)
1. Linting of the `styles-old.less` file shouldn't show any errors

### Actual result (*)
1. Linting of the `styles-old.less` file shows the following errors:
```
extend ' .data-table tbody tr:nth-child(odd) td' has no matches
extend ' .grid-actions .export .label' has no matches
extend ' .grid-actions .export .action-' has no matches
extend ' .col-570' has no matches
```

**Comment by magento-engcom-team** [comment]
@engcom-Charlie Thank you for verifying the issue.

Unfortunately, not enough information was provided to acknowledge ticket. Please consider adding the following:
- [x] Add `"Component: "` label(s) to this ticket based on verification result. If uncertain, you may follow the best guess

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Charlie 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-18940` were created

**Issue Available**: @engcom-Charlie, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @hostep. Thank you for your report. 
The issue has been fixed in magento/magento2#28895 by @mrtuvn in 2.4-develop branch
Related commit(s):
 - [e715fdcea41e821441ff17ee360b352f91f3248a](https://github.com/magento/magento2/commit/e715fdcea41e821441ff17ee360b352f91f3248a)

The fix will be available with the upcoming 2.4.1 release.
